bigbluebutton-html5/imports/ui/components/dropdown/component.jsx

```diff
@@ -122,7 +125,14 @@ class Dropdown extends Component {
   }
 
   render() {
-    const { children, className, style, intl } = this.props;
+    const {
+      children,
+      className,
+      style, intl,
+      hasPopup,
+      ariaLive,
```

Comment:
You can just pick the other props with an destructuring assignment and use them in the component.

```
const { children, className, intl, ...otherProps } = this.props;
...
return (
  <div
    className={cx(styles.dropdown, className)}
    {...otherProps}
  >
  ...
);
```

And then instead of using custom props, just use the `aria-*` by themselves.